reference/orders.v3.yml

```diff
@@ -2701,6 +2705,8 @@ components:
         * `TAX`
         * `FEE`
```

Comment:
may you pls help to update the list of "Amount Bound Item" to only these:
        * `ORDER`
        * `SHIPPING`
        * `HANDLING`
        * `TAX`
        * `FEE`